x-pack/legacy/plugins/apm/server/lib/transaction_groups/fetcher.ts

```diff
@@ -50,6 +51,37 @@ export function transactionGroupsFetcher(
     { '@timestamp': { order: 'desc' as const } }
   ];
 
+  const transactionsAggregation = {
+    transactions: {
+      terms: {
+        ...projection.body.aggs.transactions.terms,
+        order: { sum: 'desc' as const },
+        size: config['xpack.apm.ui.transactionGroupBucketSize']
+      },
+      aggs: {
+        sample: { top_hits: { size: 1, sort } },
+        avg: { avg: { field: TRANSACTION_DURATION } },
+        p95: { percentiles: { field: TRANSACTION_DURATION, percents: [95] } },
+        sum: { sum: { field: TRANSACTION_DURATION } }
+      }
+    }
+  };
+
+  const servicesAggregation = {
+    services: {
+      terms: {
+        field: SERVICE_NAME,
+        size: config['xpack.apm.ui.transactionGroupBucketSize']
+      },
+      aggs: transactionsAggregation
+    }
+  };
+
+  const isTopTraces = options.type === 'top_traces';
```

Comment:
if we have to delete it, we might want to remove it from the projection entirely? the purpose of the projection is to be able to share (parts of a) query for re-use. It might be better to create a new projection in this case for traces. I think this is pretty much what Søren was alluding to in the issue.

If we have separate projections it should also be easier to fix the issue with the counter, so probably good to wait until that discussion is resolved.

Reply:
I agree that deleting the projection aggregation isn't the best to do. But I'd rather wait for the discussion about the importance of the counter.

Reply:
@dgieselaar but besides the discussion about the counter, I can create a new projection for the trace and remove the `delete` from the fetcher. WDYT?

Reply:
@dgieselaar We decided to remove the counter on this page for now (https://github.com/elastic/kibana/issues/54250#issuecomment-572685411) do you think it is still valid to create a new projection?

Reply:
No, I don't think we nee it. Are we only going to hide the counter for this view, or for all views?

Reply:
@cauemarcondes @dgieselaar I think we should only hide it for this view, because it's the only place where the count is not correct. I have a proposed enhancement ready, but it won't make it into the next release, so I'm good with just taking it out of the Traces list for now.

Reply:
@dgieselaar @formgeist  I removed it only for the trace overview page.